db/model/collector.js

```diff
@@ -124,6 +131,23 @@ module.exports = function collector(seq, dataTypes) {
           throw new ValidationError(msg);
         }
       }, // hooks.beforeUpdate
+
+      afterUpdate(inst /* , opts */) {
+        /* if status is changed to Running, then find and assign unassigned
+         generators */
+        if (inst.changed('status')) {
+          if (inst.status === collectorStatus.Running) {
+            return u.findAndAssignGenerators(seq);
+          } else if (inst.status === collectorStatus.Stopped ||
```

Comment:
the preceding "if" block returns so shouldn't this just be a separate if rather than an "else"?

Reply:
My two cents: I generally prefer using if/else in cases like this (as Pallavi had it originally), even though it's not strictly necessary, because it's more compact and the control flow is clear at a glance.